test/functional/mempool_orphan_peer_disconnected.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2021 The Bitcoin Core developers
```

Comment:
Is this license correct?

Reply:
Uh, not sure. I started by copying another functional test so we multiple of these

Reply:
Changed this to the usual license but as I said, there still are tests with this

Reply:
For some tests where we've taken (and modified) the original Bitcoin test it probably makes sense to have the original Bitcoin license header and then "update" it to add a Mintlayer-y bit to cover the work we've added. Where it's not a test derived from Bitcoin (like this one? I couldn't see a comparable test in Bitcoin core) I guess our license alone is the way to go. It probably means some of the other files need licenses updated. I'll go through the functional tests this arvo and have a look.